wishlist/wishlist.html

```diff
@@ -0,0 +1,109 @@
+                <div class="icon-with-badge"><a href="/cart/cart.html">
+                        <span class="material-icons outlined lg">
+                            shopping_cart
+                        </span>
+                        <small class="text-center notification mat xsm">0</small></a>
+                </div>
+            </div>
+        </header><br>
+        <section class="wishlist-heading text-center bold">My Wishlist (<small
+                class="wishlist-counter size-16 text-blue bold"> 4
+            </small>items)</section>
+        <section class="wishlist-items">
+            <div class="eg-card card-one without-shadow">
+                <header class="card-header-text bold">FingerPrint Sensors<br>
+
+                </header>
+
+                <img class="card-image" src="/images/fingerPrintSensors.jpg" alt="Finger Print Sensor">
+
+                <footer class="card-footer-icons">
+                    <p class="md">$299</p>
+                    <button class="primary btn without-shadow sm">Move to Cart</button>
+                    <button class="secondary btn sm">Remove from wishlist</button>
+                </footer>
+
+            </div>
+            <div class="eg-card card-one without-shadow">
+
+                <header class="card-header-text bold">Lambo Key Cover<br>
+                </header>
+
+                <img class="card-image" src="/images/lambokey.jpg" alt="Lamborghini KeyCover">
+
+                <footer class="card-footer-icons">
+                    <p class="md">$28</p>
+
+                    <button class="primary btn without-shadow sm">Move to Cart</button>
+                    <button class="secondary btn sm">Remove from wishlist</button>
+                </footer>
+            </div>
+            <div class="fourth eg-card card-one without-shadow">
+
+                <header class="card-header-text bold">Lambo Badge<br>
+                </header>
+
+                <img class="card-image" src="/images/lamboBadge.jpg" alt="Lamborghini Badge">
+
+                <footer class="card-footer-icons">
+                    <p class="md">$5</p>
+
+                    <button class="primary btn without-shadow sm">Move to Cart</button>
+                    <button class="secondary btn sm">Remove from wishlist</button>
+                </footer>
+            </div>
+            <div class="third eg-card card-one without-shadow">
+
+                <header class="card-header-text bold">Martini Exhaust<br>
+                </header>
+
+                <img class="card-image" src="/images/martini.jpg" alt="Lamborghini Exhaust System">
+
+                <footer class="card-footer-icons">
+                    <p class="md">$250</p>
+
+                    <button class="primary btn without-shadow sm">Move to Cart</button>
+                    <button class="secondary btn sm">Remove from wishlist</button>
+                </footer>
+            </div>
+        </section>
+    </div>
+</body>
+
+</html>
```

Comment:
Just configure and run prettier once you're done with all your screens. Not required, just a suggestion.

Reply:
Yeah sure will do that. Thank you for the suggestion.